### DISCUSSION TITLE
Import Error- remappings looks fine, and also tried the normal method.

### DISCUSSION THREAD

**Discussion by Glow-in-the-dark** [Cyfrin/foundry-full-course-cu#499] [Q&A] [answered]
This is my Error.
![Screenshot 2023-07-31 at 11 08 34 PM](https://github.com/Cyfrin/foundry-full-course-f23/assets/2751458/9f21321b-02d7-47b9-be48-966d981e19d2)

my foundry.toml:
```
[profile.default]
src = "src"
out = "out"
libs = ["lib"]
remappings = ["@openzeppelin/contracts-upgradeable=lib/openzeppelin-contracts-upgradeable/contracts"]

# See more config options https://github.com/foundry-rs/foundry/tree/master/config
```

I don't understand why there is an error when the path seems to be right.
to replicate:
https://github.com/Glow-in-the-dark/Solidity-playground/tree/main/EtherJS_Foundry/foundry-upgrades-proxy
this is my repo.

Many thanks.

**Comment by shikhar229169** [comment, marked as answer]
```solidity
import {UUPSUpgradeable} from "lib/openzeppelin-contracts-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol";
```

This would work.
And for that remappings, I also got the same error. And its only happening with the openzeppelin-contracts-upgradeable library.

**Reply by Glow-in-the-dark** [reply]
Ahh thanks Shikar !!! weirdly, my remapping work now, and i have no idea why. I didn't changed anything. Merely uncommented it. 
Thanks alot btw Shikar!!
